bin/setup

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
```

Comment:
Do we want to explain this part of the setup?

Reply:
Actually, it is not necessary for the setup process at this implementation. 
However It kept this part according to the setup generated by `bundle gem`, so it is the same as gem generated by `bundle gem` and I think that explanation is unnecessary.

If it is better not to write it as it is unnecessary processing, I will remove this part.

Reply:
Got it. OK, let's keep it then. Thanks for the explanations!